.github/dependabot.yml

```
@@ -17,7 +17,8 @@ updates:
       - "javascript"
       - "type::chore"
     schedule:
-      interval: "weekly"
+      interval: "monthly"
+    open-pull-requests-limit: 5  
```

5 is too few

we should set something like 25 IMO
```suggestion
    open-pull-requests-limit: 25
```